lib/uncruft.rb

```diff
@@ -10,11 +13,11 @@ class << self
     FALSE_VALUES = [false, 0, "0", "f", "F", "false", "FALSE", "off", "OFF"].to_set
 
     def record_deprecations?
-      ENV['RECORD_DEPRECATIONS'].presence && !FALSE_VALUES.include?(ENV['RECORD_DEPRECATIONS'])
+      ENV['RECORD_DEPRECATIONS'].presence && !ENV['RECORD_DEPRECATIONS'].in?(FALSE_VALUES)
```

Comment:
Rubocop wanted me to change this to `!FALSE_VALUES.exclude?(val)` but the `Set` type doesn't have that method. 🤔 

So I went with `!val.in?(FALSE_VALUES)` instead 🧠

Reply:
ugh I hate it when rubocop is wrong like this